external-crates/move/crates/move-analyzer/prettier-plugin/src/printer.ts

```diff
@@ -71,11 +72,13 @@ export function print(path: AstPath, options: ParserOptions, print: printFn) {
             //     f2: u64,
             // }
             return [
+                node.child(0).type == 'public' ? 'public ' : '',
```

Comment:
nit: I believe `===` is stylistically preferred in TS code. I.e., if you _can_ use `===`, do. I searched our sui repo and we seem to be doing that for the most part, there are some `==` here and there